src/h5web/providers/utils.ts

```diff
@@ -36,6 +37,10 @@ export function isSimpleShape(shape: HDF5Shape): shape is HDF5SimpleShape {
   return shape.class === HDF5ShapeClass.Simple;
 }
 
+export function isScalarShape(shape: HDF5Shape): shape is HDF5ScalarShape {
+  return shape.class === HDF5ShapeClass.Scalar;
+}
+
```

Comment:
The naming pains me a bit here. It is not the shape that is Scalar but rather the value it describes. Anyway, I think this is still the best solution to be consistent with `ShapeClass` and the rest of the code.